packages/x-components/src/x-modules/tagging/store/emitters.ts

```diff
@@ -6,4 +6,9 @@ import { taggingXStoreModule } from './module';
  *
  * @internal
  */
-export const taggingEmitters = createStoreEmitters(taggingXStoreModule, {});
+export const taggingEmitters = createStoreEmitters(taggingXStoreModule, {
+  ConsentChanged: {
+    selector: state => state.consent!,
+    filter: consent => !consent
```

Comment:
Hmm, I think this event is lying, it says that the consent has changed, and by that I understand to any possible value, but in reality, it will only be emitted when the consent is false. Maybe we can do that `filter` in the wire instead?